docs/code-reviewing.md

```diff
@@ -0,0 +1,99 @@
+# Reviewing Zulip server code
+
+This document is a brief discussion of what we look for when reviewing
+contributions to Zulip. It's meant partially for developers who want
+to get their code merged faster, and partially for developers who have
+made successful pull requests already and would like to start
+participating in code review.
+
+* *The Travis CI build.* The tests need to pass. One can investigate
+  any failures and figure out what to fix by clicking on a red X next
+  to the commit hash or the Detail links on a pull request. (Example:
+  in [#1219](https://github.com/zulip/zulip/pull/1219), click the red
+  X next to `f1f474e` to see the build jobs for that commit, at least
+  one of which has failed. Click on the link for Travis continuous
+  integrations details to see [the tests Travis ran on that
+  commit](https://travis-ci.org/zulip/zulip/builds/144300899), at
+  least one of which failed, and go to [one of the failing
+  tests](https://travis-ci.org/zulip/zulip/jobs/144300901) to see the
+  error.)
```

Comment:
Can you add a note that Coveralls' data on changes isn't always accurate, so one should look at the test coverage situation directly?